website/content/docs/commands/operator/init.mdx

```diff
@@ -43,12 +43,12 @@ $ vault operator init \
     -pgp-keys="keybase:hashicorp,keybase:jefferai,keybase:sethvargo"
 ```
 
-Initialize Auto Unseal, but encrypt the recovery keys with pgp keys:
+Initialize Auto Unseal with a non-default threshold and number of recovery keys, and encrypt the recovery keys with pgp keys:
 
 ```shell-session
 $ vault operator init \
-    -recovery-shares=1 \
-    -recovery-threshold=1 \
+    -recovery-shares=7 \
+    -recovery-threshold=4 \
     -recovery-pgp-keys="keybase:grahamhashicorp"
```

Comment:
Since we're setting the shares to be `7`, we have to update the PGP keys as well:

```suggestion
    -recovery-pgp-keys="keybase:jeff, keybase:chris, keybase:brian, keybase:calvin, keybase:matthew, keybase:vishal, keybase:nick"
```